valhalla/baldr/traffictile.h

```diff
@@ -21,70 +21,78 @@
 #ifndef C_ONLY_INTERFACE
 namespace valhalla {
 namespace baldr {
-namespace traffic {
 
 using std::uint16_t;
 using std::uint32_t;
 using std::uint64_t;
 #endif
 
-constexpr uint16_t INVALID_SPEED_AGE_BUCKET = 0;
-constexpr uint16_t MAX_SPEED_AGE_BUCKET = 15;
-constexpr uint16_t SPEED_AGE_BUCKET_SIZE = 2; // 2 minutes per bucket
+// Traffic speeds are encoded as 7 bits in `TrafficSpeed` below with a 2kph multiplier
+constexpr uint32_t MAX_TRAFFIC_SPEED_KPH = 252;
```

Comment:
i think you can keep this at 254 if you use the breakpoint1==0 to specify invalid right?

Reply:
Still need a value to signal that a speed is unknown. E.g. the case of

```
| ... unknown speed ... | ... known speed... | .. |
```

Reply:
if the speed is unknown how can their be a breakpoint1 taht is not 0? are you saying you can have congestion but no speed? that seems nonsensical

Reply:
oh i see, its probably more common where you have `closed | unknown | unknown`. ignore me :D